bibimbap/templates/404.html

```diff
@@ -0,0 +1,17 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+
+<!-- wp:group {"tagName":"main","style":{"spacing":{"blockGap":"var:preset|spacing|50","margin":{"top":"var:preset|spacing|80","bottom":"var:preset|spacing|60"},"padding":{"top":"24vh","right":"var:preset|spacing|50","bottom":"24vh","left":"var:preset|spacing|50"}}},"layout":{"type":"constrained"}} -->
+<main class="wp-block-group" style="margin-top:var(--wp--preset--spacing--80);margin-bottom:var(--wp--preset--spacing--60);padding-top:24vh;padding-right:var(--wp--preset--spacing--50);padding-bottom:24vh;padding-left:var(--wp--preset--spacing--50)"><!-- wp:heading {"textAlign":"left","level":1,"align":"wide","style":{"typography":{"fontSize":"72px"}}} -->
+<h1 class="alignwide has-text-align-left" style="font-size:72px">Oops! That page can’t be found.</h1>
+<!-- /wp:heading -->
+
+<!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
+<div class="wp-block-group alignwide"><!-- wp:paragraph -->
+<p>It looks like nothing was found at this location. Maybe try a search?</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:search {"label":"","showLabel":false,"style":{"border":{"radius":"2px"}}} /--></div>
+<!-- /wp:group --></main>
+<!-- /wp:group -->
+
+<!-- wp:template-part {"slug":"footer","tagName":"footer"} /-->
```

Comment:
Should this file be using the 404 pattern?

Reply:
Yeah. updated the template.